### DISCUSSION TITLE
Support for AliCloud

### DISCUSSION THREAD

**Discussion by hardy4yooz** [infracost/infracost#1502] [Ideas & Feature Requests]
This is a cool tool, will you consider supporting [AliCloud](https://www.aliyun.com/).
I am willing to contribute some code.

> Please vote on this issue by adding a 👍 reaction to help the community and maintainers gauge interest.

**Comment by tim775** [comment]
Hi @hardy4yooz, thanks for the offer to help! So far we've focused on the three big cloud providers (AWS, Azure and Google). Here's the rough process we followed for those:

1. Add support for all prices we could get from the cloud vendor.
2. Added support for a few Terraform resources to the Infracost CLI - this demonstrates that the prices have been added in a way that the CLI can use them. Tests can be written for those few resources (in the CLI repo) that show the cost estimates are correct.
3. Ask the community to help with adding more resources for that cloud vendor.

We have seen much interest in AliCloud from the community yet, but I assume you're an AliCloud user.  If so, and you have time to do steps 1 and 2 then join the community slack (https://infracost.io/community-chat) and we can discuss first steps there.
